diagnostic.md

```diff
@@ -36,13 +41,19 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/$ID
 ```
```

Comment:
$ID is not defined. We don't want to assume we have access to that variable, so for this example we want you to hardcode that yourself `curl --include --request GET http://www.example.com/donuts/42`